npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater_spec.rb

```diff
@@ -2085,20 +2125,30 @@
           end
 
           it "updates the right file" do
-            root_lockfile = updated_files.find { |f| f.name == "package-lock.json" }
-            parsed_root_lockfile = JSON.parse(root_lockfile.content)
+            updated_npm_lock_content = updated_files.find { |f| f.name == "package-lock.json" }
+            expected_updated_npm_lock_content = fixture(
+              "updated_projects", "npm7", "workspaces_dev", "package-lock.json"
+            )
+            parsed_npm_lockfile = JSON.parse(updated_npm_lock_content.content)
             expect(updated_files.map(&:name)).
-              to match_array(%w(package-lock.json packages/package1/package.json))
-            expect(parsed_root_lockfile.dig("dependencies", "etag", "version")).to eq("1.8.1")
+              to match_array(%w(package-lock.json other_package/package.json packages/package1/package.json))
+            expect(parsed_npm_lockfile.dig("dependencies", "etag", "version")).to eq("1.8.1")
+            expect(updated_npm_lock.content).to eq(expected_updated_npm_lock_content)
```

Comment:
added an expected updated fixture to make this spec a bit more robust, created it by manually updating the nested package.json requirement and running `npm install` in the root folder

Reply:
FYI this was previously adding `etag` as a top-level dependency but wasn't covered by this spec